memory/src/main/scala/filodb.memory/format/BinaryVector.scala

```diff
@@ -176,6 +176,11 @@ trait CounterVectorReader extends VectorDataReader {
    * @param meta CorrectionMeta with total running correction info.  lastValue is ignored
    */
   def updateCorrection(acc: MemoryReader, vector: BinaryVectorPtr, meta: CorrectionMeta): CorrectionMeta
+
+  /**
+    * Identifies row numbers at which histogram or counter has dropped
+    */
+  def dropPositions(acc2: MemoryReader, vector: BinaryVectorPtr): debox.Buffer[Int]
```

Comment:
Can we have the resets function implementation use this method?

Reply:
We should eventually use it, outside scope of this PR. Not a simple change because drops across chunks is not covered with this helper.